website/client/src/components/shared/closeIcon.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <button title="close dialog">
+    <svg height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"
```

Comment:
can we import the svg like we usually do for icons? it'll make sure the code is transformed by svgo and more consistent

Reply:
that was the issue we chatted about, when using this svg, it displayed (for example) unequip.svg, maybe that was random, but it was never this new close svg , thats why I copied the svg-source, until the svgo works fine again

Reply:
if I remember correctly @Tressley said that the previous close icon file is totally fine and there's no reason to use the different one

Reply:
used the current close icon on it